src/dotnet-svcutil/lib/src/FrameworkFork/Microsoft.Xml/External/Evidence.cs

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+namespace Microsoft.Xml.Serialization {
+				using System;
+				using Microsoft.Xml;
+    internal class Evidence
+    {
```

Comment:
Remove this class. It was used for CAS which doesn't exist in core and is deprecated in NetFx. It's only used in TempAssembly so remove the usage from that constructor.